### ISSUE TITLE
2D array

### ISSUE THREAD

**Issue by gauthammuppalla** [mikolajmitura/java-properties-to-json#45]
2D arrays are not being parsed properly

**Comment by gauthammuppalla** [comment]
pl.jalokim.propertiestojson.util.exception.ParsePropertiesException: Current array element: 'coordinates[0]' is already considered as object JSON Element not as primitive because already this field has value:

This is the error I'm getting

**Comment by mikolajmitura** [comment]
I suppose the problem is like below:
```properties
array[0][0]=test00
array[0][1]=test01
array[0][2]=test02
array[1][0]=test10
array[1][1]=test11
```

expected value:
```json
{  
   "array":[  
      [  
         "test00",
         "test01",
         "test02"
      ],
      [  
         "test10",
         "test11"
      ]
   ]
}
```

**Comment by gauthammuppalla** [comment]
Yes, is this functionality available already?

**Comment by mikolajmitura** [comment]
not yet... ;/

**Comment by gauthammuppalla** [comment]
Could you attach a small ReadMe file about the current code framework which I could go through, to try and add this feature?

**Comment by mikolajmitura** [comment]
I'm working on that at this moment. 
Now you can try with code from branch fature/multi_dim_array, but multi dimensional array should work.
But not all tests pass correctly now, you can skip them or ignore. 
If I resolve all problems then I will release new version. 
And thanks for you issue :-)

**Comment by mikolajmitura** [comment]
this issue resolved in 5.0.0 version.
Should be available through in few hours at central maven repository.
